.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+# this is a simple docker compose that has been tested to work with remote containers in VS code.
+# You can run these samples in the browser with no need to run Temporal Server locally
+# by using Codespaces on github. 
+# see https://github.com/temporalio/docker-compose for other reference docker-compose files
+version: '3.5'
+services:
+  ## uncomment if you need search attributes feature
+  ## dont forget to uncomment the ES related comments in the temporal service below
+  # elasticsearch:
+  #   container_name: temporal-elasticsearch
+  #   environment:
+  #     - cluster.routing.allocation.disk.threshold_enabled=true
+  #     - cluster.routing.allocation.disk.watermark.low=512mb
+  #     - cluster.routing.allocation.disk.watermark.high=256mb
+  #     - cluster.routing.allocation.disk.watermark.flood_stage=128mb
+  #     - discovery.type=single-node
+  #     - ES_JAVA_OPTS=-Xms100m -Xmx100m
+  #   image: elasticsearch:7.10.1
+  #   # networks:
+  #     # - temporal-network
+  #   ports:
+  #     - 9200:9200
+  devcontainer:
+    container_name: devcontainer
+    depends_on:
+      - temporal
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        VARIANT: "1-bullseye"
+        NODE_VERSION: "none"
+    volumes:
+      # This is where VS Code should expect to find your project's source code and the value of "workspaceFolder" in .devcontainer/devcontainer.json
+      - ..:/workspace:cached
+    command: /bin/sh -c "while sleep 1000; do :; done"  
+    # Runs app on the same network as the service container, allows "forwardPorts" in devcontainer.json function.
+    network_mode: service:temporal
+  postgresql:
+    container_name: temporal-postgresql
+    environment:
+      POSTGRES_PASSWORD: temporal
+      POSTGRES_USER: temporal
+    image: postgres:13
+    ports:
+      - 5432:5432
+  temporal:
+    container_name: temporal
+    depends_on:
+      - postgresql
+    #  - elasticsearch
+    environment:
+      - DB=postgresql
+      - DB_PORT=5432
+      - POSTGRES_USER=temporal
+      - POSTGRES_PWD=temporal
+      - POSTGRES_SEEDS=postgresql
+      - DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development_es.yaml
+    #  - ENABLE_ES=true
+    #  - ES_SEEDS=elasticsearch
+    #  - ES_VERSION=v7
+    image: temporalio/auto-setup:1.13.4
```

Comment:
This is an old version. Maybe `temporalio/auto-setup:latest` would be better (same with other Temporal images herein).

Reply:
🤦 sorry about that. latest is definitely better.